lib/taurus/core/tango/tangovalidator.py

```diff
@@ -52,7 +52,10 @@ class TangoAuthorityNameValidator(TaurusAuthorityNameValidator):
     '''
 
     scheme = 'tango'
-    authority = '//(?P<host>([\w\-_]+\.)*[\w\-_]+):(?P<port>\d{1,5})'
+    authority = '//((?P<host>([\w\-_]+\.)*[\w\-_]+):(?P<port>\d{1,5})' \
```

Comment:
please do not use implicit concatenation of strings (i.e. use parentheses and + operators instead of `\`)